### ISSUE TITLE
still very dangerous one-key shortcuts # and x

### ISSUE THREAD

**Issue by rubo77** [willsALMANJ/tbkeys#40]
I think the main purpose of this add on is to unset the dangerous keys, that cause havoc if you type them uncontrolled because you happen to have the focus on the wrong window, so those new shortcuts should be **optional** for users that are not afraid of those single-key-shortcuts:

-  "#": "cmd:cmd_delete",
-  "x": "cmd:cmd_archive",

the others are harmless and just open some windows ;)

**Comment by tvercaut** [comment]
Adding an FAQ including typical configs would be great for newcomers. I was looking for a means of replaing the "a" shortcut for something less error prone (following the loss of several previous add-ons after being upgraded to TB78) and am glad I found tbkeys-lite!

For now, this is what I used to match the archive shortcut from outlook on Mac OS (which I also have to use from time to time):
```json
{
    "a": "unset",
    "ctrl+e": "cmd:cmd_archive"
}
```

I guess I will now also start unsetting other one-key shortcuts which have been bothering me since keyconfig stopped being supported.

**Comment by rubo77** [comment]
> ... that the default install should set no key bindings.

yes, but not just set **no config**, better set each config for the keys, that they were without this plugin, for example set expliciltey "a" to "cmd:cmd_archive", so you see intuitively, what you need if you want to move all one-key-shortcuts to something else.

**Comment by willsALMANJ** [comment]
Hmm, maybe those could be optionally populated by a button or at least documented on the wiki. My concern with populating them by default is that the way the "unset singles" button works now is that it only unsets the keys that do not currently have a shortcut set. I like it working this way to avoid someone accidentally overwriting their settings.
